src/applications/gqrx/remote_control.cpp

```diff
@@ -429,19 +429,24 @@ void RemoteControl::setMode(int mode)
 void RemoteControl::setNewRemoteFreq(qint64 freq)
 {
     qint64 delta = freq - rc_freq;
+    // it's an ugly but I hope this will be fixed with the issue #218
+    int FilterBW_half = std::min<int>(bw_half, 100000 /* WFM Wide */);
```

Comment:
Sorry, but this doesn't work well with narrow band modes or when the total bandwidth is few 100 kHz.